pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[build-system]
+requires = ["setuptools >= 40.6.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[tool.pytest.ini_options]
+testpaths = [
+    "tests",
+]
+# We're working towards getting all the tests working, but so far these are
+# the ones that dont work with the pip installation:
+addopts = """
+--ignore=tests/test_bleed_trails.py
+--ignore=tests/test_skyModel.py
+--ignore=tests/test_camera_readout.py
+--ignore=tests/test_get_stack_products.py
+--ignore=tests/test_flats.py
+--ignore=tests/test_imageSimulator.py
+--ignore=tests/test_raw_file_writing.py
```

Comment:
Hi @g-braeunlich,
Could you add
```
--ignore=tests/test_skycat.py
```
here please?   Fixing this test to use the `pathlib` stuff is a bit more complicated because there is a path in `tests/data/sky_cat_9683.yaml` that the skyCatalogs code uses that can't make use of this mechanism, so I think some changes in skyCatalogs need to be made also.

Reply:
If some function is not `pathlib.Path` aware, it should be in principle as simple as converting the `Path` obj to a `str`.
I tried that and pushed. If this does not work, I ll have another look tomorrow. Sorry, that I forgot to watch out for new commits!

Reply:
Also had to change the path
`root_directory: ../tests/data`
to
`root_directory: ./tests/data`

in [tests/data/sky_cat_9683.yaml](tests/data/sky_cat_9683.yaml)

Reply:
Thanks, that will work of course for the GHA testing.  I've been discussing with Joanne changes to skyCatalogs itself regarding how that `root_directory` entry would be handled so that one could  run `pytest` either from the top-level `imSim` folder or from the `tests` subfolder.